operations/utility_storage/Troubleshoot_Pools_Have_Many_More_Objects_Per_Pg_Than_Average.md

```diff
@@ -0,0 +1,47 @@
+# Fixing incorrect number of PG Issues
+
+## Scenario 1
+
+**Symptom:** Ceph is reporting a `HEALTH_WARN` and the warning message is `1 pools have many more objects per pg than average`.
+
+**Cause:**
+
+When an additional pool to the store the CSM artifacts was introduced, it caused the amount of PGs created per OSD to not allow for the amount of data stored in that device.
+This will typically be seen on three node Ceph clusters since the number of PGs is set by the number of OSDs.
+
+**Fix:**
+
+1. Get the current `pg_num` and `pgp_num` value for the pool that is warning.
+
+   ```bash
+   ceph osd pool get csm_admin_pool pg_num
+   ceph osd pool get csm_admin_pool pgp_num
+
+   ```
+
+   Example Output:
+
+   ```screen
+   pg_num: 32
+   pgp_num: 32
+   ```
+
+2. Set the new values.
+
+   ```bash
+   ceph osd pool set csm_admin_pool pg_num 64
+   ceph osd pool set csm_admin_pool pgp_num 128
+   ceph osd pool set csm_admin_pool  pg_num_min 64
+   ceph osd pool set csm_admin_pool  pg_num_max 128
```

Comment:
Also where do the magic values of 64/128 come from?

Reply:
What this does is set some of our default values to ensure that we have enough pgs in the pool for the massive amount of files we are shoving into that 1TiB volume.   I wasn't setting these on pool creation like the other pools